documentation/SOFTWARE_REQUIREMENTS_SPEC.md

```diff
@@ -14,12 +14,12 @@ Our web-based Sokoban game will have greater user reachability compared to a mor
 
 ### Phase 1: Basic Gameplay Requirements
 
-#### World
+#### Game Board
 
-* The world shall be approximately 500 pixels tall by 600 pixels wide.
-* The world shall have top view perspective (viewed from above). 
-* Every element in the world shall have a graphical dimensions of 20 pixels by 20 pixels.
-* The world shall contain the following:
+* The board shall be approximately 500 pixels tall by 600 pixels wide.
+* The board shall have top view perspective (viewed from above). 
+* Every element in the board shall have a graphical dimensions of 20 pixels by 20 pixels.
```

Comment:
We might want to leave this up in the air or allow for responsiveness/mobile view as mentioned above.

Reply:
Then, does that mean the web browser will adjust for the monitor resolution or will it be like steam where we can have a default resolution when a window opens up?

Reply:
I was just thinking of default dimensions for the graphic designer. But maybe this is not the right document for that. I know that any size of the image can be adjusted  through a stylesheet but it gives the designers an idea of how big our images and shape (square vs rectangular).

Reply:
```suggestion
* Every element in the board shall have a square graphical dimension.
```